TeamSnapSDK/SDK/DataLayer/TSDKDataRequest.m

```diff
@@ -112,6 +112,7 @@ + (NSDictionary *)requestHeaders {
         [mutableRequestHeaders setObject:@"iOS" forKey:@"X-Client-Source"];
         [mutableRequestHeaders setObject:@"3" forKey:@"X_TEAMSNAP_API_VERSION"];
         [mutableRequestHeaders setObject:self.deviceInfo forKey:@"User-Agent"];
+        [mutableRequestHeaders setObject:@"Teams iOS" forKey:@"X-TS-Client-Name"];
```

Comment:
FWIW, the openapi header for this is called `x-ts-platform` and passes `ios` should we keep consistent? Or should we say `TSDK-iOS` to differentiate?

Reply:
I changed it to `x-ts-client-name`